index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <link rel="stylesheet" href="index.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Profile</title>
+</head>
+
+<body>
+    <nav id="nav">
+        <div class="nav">
+            <a class="nav__logo" href="#">My Logo</a>
+
+            <img class="nav__hambuger" src="images/hambuger.svg" alt="Icon">
+
+        </div>
+    </nav>
+
+    <section id="hero">
+        <div class="hero">
+            <h1 class="hero__heading">
+                I’m Najash
+                Glad to see you!
+            </h1>
+            <p class="hero__desc">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my
+                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.
+            </p>
+            <p class="hero__subtitle">
+                LET’S CONNECT
+            </p>
```

Comment:
- [ ] [OPTIONAL] It would be better if you used a `<a>` tag for this link so you can refer to an email or external website